great_expectations/exceptions/resource_freshness.py

```diff
@@ -0,0 +1,96 @@
+"""
+This module contains exceptions related to resource freshness.
+
+A resource is considered "fresh" if it has been added to the DataContext and is up-to-date with
+its persisted equivalent.
+
+Please see the `FreshnessDiagnostics` class for how these exceptions are aggregated and raised.
+"""
```

Comment:
You could add the hierarchy here so someone adding one sees the intended structure.